mscore/scoretab.cpp

```diff
@@ -53,13 +55,17 @@ ScoreTab::ScoreTab(QList<MasterScore*>* sl, QWidget* parent)
       connect(ag, SIGNAL(triggered(QAction*)), this, SIGNAL(actionTriggered(QAction*)));
 
       tab = new QTabBar(this);
+      tab->setObjectName("primary tab");
```

Comment:
I am guessing you probably did not mean to include a space in the object name. I don't know if it matters or not, but no other object name contains any spaces.

Reply:
Thanks, fixed

Reply:
I still see the space though